verilog/tools/kythe/kythe_facts_extractor.h

```diff
@@ -0,0 +1,157 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERIBLE_VERILOG_TOOLS_KYTHE_KYTHE_FACTS_EXTRACTOR_H_
+#define VERIBLE_VERILOG_TOOLS_KYTHE_KYTHE_FACTS_EXTRACTOR_H_
+
+#include <string>
+#include <utility>
+
+#include "verilog/tools/kythe/indexing_facts_tree.h"
+#include "verilog/tools/kythe/indexing_facts_tree_extractor.h"
+
+namespace verilog {
+namespace kythe {
+
+namespace {
```

Comment:
Only use anonymous namespaces in .cc files, not .h files, or it may cause linker issues.
If you need a private namespace, one convention is to name it `internal`, which means, only your .cc file is allowed to reference `internal` symbols.